src/main/resources/application.conf

```diff
@@ -136,7 +136,6 @@ database {
       db.url = "jdbc:hsqldb:mem:${slick.uniqueSchema};shutdown=false;hsqldb.tx=mvcc"
       db.driver = "org.hsqldb.jdbcDriver"
       driver = "slick.driver.HsqldbDriver$"
-      slick.createSchema = true
```

Comment:
The ctmpl file was for the server, that didn't use the old schema creator, and wasn't present [as far as I saw](https://github.com/broadinstitute/cromwell/blob/10f8d14842993f29c7e2e20a05639576509eda9a/src/main/config/cromwell.conf.ctmpl#L41-L48).